### ISSUE TITLE
Tox doesn't have a Python 3.5 environment

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8106]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)| @hawkowl reported|
|-|-|
|Trac ID|trac#8106|
|Type|enhancement|
|Created|2015-11-15 10:15:08Z|
|Branch|https://github.com/twisted/twisted/tree/py35-tox-8106|

It should have one, it's pretty important.

<details><summary>Searchable metadata</summary>

```
trac-id__8106 8106
type__enhancement enhancement
reporter__hawkowl hawkowl
priority__normal normal
milestone__Python_3_x Python-3.x
branch__branches_py35_tox_8106 branches/py35-tox-8106
branch_author__hawkowl hawkowl
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1447582508476290 1447582508476290
changetime__1447814176333203 1447814176333203
version__None None
owner__hawkowl hawkowl

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_1"></a>|@hawkowl commented|
|-|-|

(In [46218]) Branching to py35-tox-8106.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_2"></a>|@hawkowl commented|
|-|-|

This makes coverage work again, adds Python 3.5, and makes it so that you can use Python 3.5 as the interpreter running ```tox```.

Builders spun, please review.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban set owner to @hawkowl|
|-|-|

Changes looks good. Thanks!

Looking forward for integrating tox with our builders

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_4"></a>|@hawkowl set status to `closed`|
|-|-|

(In [46242]) Merge py35-tox-8106: Add Python 3.5 to the Tox matrix

Author: hawkowl
Reviewer: adiroiban
Fixes: [#8106](https://github.com/twisted/twisted/issues/8106)
